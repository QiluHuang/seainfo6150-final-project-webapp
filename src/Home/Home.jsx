import React from 'react'
import Form from "../Form/Form";

const Home = () => {
    return (
        <div>
            Uncle Wong's
            <h1>I changed this homepage</h1>
        </div>
    )
}

export default Home
