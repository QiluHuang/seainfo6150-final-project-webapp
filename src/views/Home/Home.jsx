import React, { useState } from 'react';
import styles from './Home.module.css';

import Slider from '../../components/Slider/Slider.jsx';


const Home = () => {
    const [isTextShowing, setIsTextShowing] = useState(false);
  
    function onClick() {
      setIsTextShowing(!isTextShowing);
    }
      return (
        <div className={styles.home_container}><Slider /></div>
      
      );
  }
  
  export default Home