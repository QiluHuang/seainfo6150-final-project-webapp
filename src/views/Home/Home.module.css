.home_containter {
    margin: 0;
    padding: 0;
}