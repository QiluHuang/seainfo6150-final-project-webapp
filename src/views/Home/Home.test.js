import React from "react";
import { render } from "@testing-library/react";
import Home from "./Home.jsx";

describe("Home tests", () => {
    it("renders the home component correctly", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
    });
});