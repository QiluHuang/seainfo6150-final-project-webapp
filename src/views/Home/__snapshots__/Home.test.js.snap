// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home tests renders the home component correctly 1`] = `
<div>
  <div
    class="home_container"
  >
    <div
      class="slider"
    >
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="barbecue-pork-fried-rice.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="chicken_with _sweet_cream_corn_soup.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="pot-stickers.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="seafood-with-tofu-soup.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="wonton-soup.jpg"
          />
        </div>
      </div>
      <button
        class="goLeft"
      >
        <i
          class="fas fa-chevron-left fa-5x"
        />
      </button>
      <button
        class="goRight"
      >
        <i
          class="fas fa-chevron-right fa-5x"
        />
      </button>
    </div>
  </div>
</div>
`;

exports[`renders the home component correctly 1`] = `
<div>
  <div
    class="home_container"
  >
    <div
      class="slider"
    >
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="barbecue-pork-fried-rice.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="chicken_with _sweet_cream_corn_soup.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="pot-stickers.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="seafood-with-tofu-soup.jpg"
          />
        </div>
      </div>
      <div
        class="slide"
        style="transform: translateX(0%);"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="slide-img"
            class="image"
            src="wonton-soup.jpg"
          />
        </div>
      </div>
      <button
        class="goLeft"
      >
        <i
          class="fas fa-chevron-left fa-5x"
        />
      </button>
      <button
        class="goRight"
      >
        <i
          class="fas fa-chevron-right fa-5x"
        />
      </button>
    </div>
  </div>
</div>
`;
